hostname sb006.cluster
ipconfig 1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000 link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00 inet 127.0.0.1/8 scope host lo valid_lft forever preferred_lft forever inet6 ::1/128 scope host valid_lft forever preferred_lft forever 2: enp4s0f0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9000 qdisc mq state UP group default qlen 1000 link/ether a0:8c:f8:61:07:9a brd ff:ff:ff:ff:ff:ff inet 10.3.81.86/22 brd 10.3.83.255 scope global enp4s0f0 valid_lft forever preferred_lft forever inet6 fe80::a28c:f8ff:fe61:79a/64 scope link valid_lft forever preferred_lft forever 3: enp4s0f1: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000 link/ether a0:8c:f8:61:07:9b brd ff:ff:ff:ff:ff:ff 4: enp4s0f2: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000 link/ether a0:8c:f8:61:07:9c brd ff:ff:ff:ff:ff:ff 5: enp4s0f3: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000 link/ether a0:8c:f8:61:07:9d brd ff:ff:ff:ff:ff:ff 6: ib0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 4092 qdisc pfifo_fast state UP group default qlen 256 link/infiniband 80:00:00:29:fe:80:00:00:00:00:00:00:88:66:39:03:00:23:be:91 brd 00:ff:ff:ff:ff:12:40:1b:ff:ff:00:00:00:00:00:00:ff:ff:ff:ff inet 10.3.89.86/22 brd 10.3.91.255 scope global ib0 valid_lft forever preferred_lft forever inet 10.3.97.86/22 brd 10.3.99.255 scope global ib0:1 valid_lft forever preferred_lft forever inet6 fe80::8a66:3903:23:be91/64 scope link valid_lft forever preferred_lft forever 7: ib1: <BROADCAST,MULTICAST> mtu 4092 qdisc noop state DOWN group default qlen 256 link/infiniband 80:00:00:29:fe:80:00:00:00:00:00:00:88:66:39:03:00:23:be:92 brd 00:ff:ff:ff:ff:12:40:1b:ff:ff:00:00:00:00:00:00:ff:ff:ff:ff
2024-02-01 08:16:13,504 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.3.89.86:46402'
2024-02-01 08:16:15,619 - distributed.diskutils - INFO - Found stale lock file and directory '/scratch/dask-worker-space/worker-r461ce4b', purging
2024-02-01 08:16:15,620 - distributed.diskutils - INFO - Found stale lock file and directory '/scratch/dask-worker-space/worker-ybr4i_hn', purging
2024-02-01 08:16:19,667 - distributed.worker - INFO -       Start worker at:     tcp://10.3.89.86:42562
2024-02-01 08:16:19,668 - distributed.worker - INFO -          Listening to:     tcp://10.3.89.86:42562
2024-02-01 08:16:19,668 - distributed.worker - INFO -           Worker name:           SLURMCluster-358
2024-02-01 08:16:19,668 - distributed.worker - INFO -          dashboard at:           10.3.89.86:42500
2024-02-01 08:16:19,668 - distributed.worker - INFO - Waiting to connect to:     tcp://10.3.88.11:37020
2024-02-01 08:16:19,668 - distributed.worker - INFO - -------------------------------------------------
2024-02-01 08:16:19,669 - distributed.worker - INFO -               Threads:                          2
2024-02-01 08:16:19,669 - distributed.worker - INFO -                Memory:                  20.49 GiB
2024-02-01 08:16:19,669 - distributed.worker - INFO -       Local Directory: /scratch/dask-worker-space/worker-j2nnmdvv
2024-02-01 08:16:19,669 - distributed.worker - INFO - -------------------------------------------------
2024-02-01 08:16:19,716 - distributed.worker - INFO -         Registered to:     tcp://10.3.88.11:37020
2024-02-01 08:16:19,716 - distributed.worker - INFO - -------------------------------------------------
2024-02-01 08:16:19,717 - distributed.core - INFO - Starting established connection to tcp://10.3.88.11:37020
2024-02-01 08:17:44,596 - distributed.core - INFO - Event loop was unresponsive in Worker for 5.25s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 08:35:11,355 - distributed.core - INFO - Event loop was unresponsive in Worker for 3.32s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 08:35:38,028 - distributed.worker - ERROR - Worker stream died during communication: tcp://10.3.89.95:36040
Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/tcp.py", line 225, in read
    frames_nbytes = await stream.read_bytes(fmt_size)
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 490, in wait_for
    return fut.result()
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 328, in connect
    handshake = await asyncio.wait_for(comm.read(), time_left())
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 492, in wait_for
    raise exceptions.TimeoutError() from exc
asyncio.exceptions.TimeoutError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2081, in gather_dep
    response = await get_data_from_worker(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2900, in get_data_from_worker
    comm = await rpc.connect(worker)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1468, in connect
    return await connect_attempt
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1389, in _connect
    comm = await connect(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 333, in connect
    raise OSError(
OSError: Timed out during handshake while connecting to tcp://10.3.89.95:36040 after 30 s
2024-02-01 08:36:08,076 - distributed.worker - ERROR - Worker stream died during communication: tcp://10.3.89.93:44896
Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/tcp.py", line 225, in read
    frames_nbytes = await stream.read_bytes(fmt_size)
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 490, in wait_for
    return fut.result()
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 328, in connect
    handshake = await asyncio.wait_for(comm.read(), time_left())
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 492, in wait_for
    raise exceptions.TimeoutError() from exc
asyncio.exceptions.TimeoutError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2081, in gather_dep
    response = await get_data_from_worker(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2900, in get_data_from_worker
    comm = await rpc.connect(worker)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1468, in connect
    return await connect_attempt
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1389, in _connect
    comm = await connect(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 333, in connect
    raise OSError(
OSError: Timed out during handshake while connecting to tcp://10.3.89.93:44896 after 30 s
2024-02-01 08:36:39,721 - distributed.worker - ERROR - Worker stream died during communication: tcp://10.3.89.95:36040
Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/tcp.py", line 225, in read
    frames_nbytes = await stream.read_bytes(fmt_size)
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 490, in wait_for
    return fut.result()
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 328, in connect
    handshake = await asyncio.wait_for(comm.read(), time_left())
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 492, in wait_for
    raise exceptions.TimeoutError() from exc
asyncio.exceptions.TimeoutError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2081, in gather_dep
    response = await get_data_from_worker(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2900, in get_data_from_worker
    comm = await rpc.connect(worker)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1468, in connect
    return await connect_attempt
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1389, in _connect
    comm = await connect(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 333, in connect
    raise OSError(
OSError: Timed out during handshake while connecting to tcp://10.3.89.95:36040 after 30 s
2024-02-01 08:36:58,071 - distributed.core - INFO - Event loop was unresponsive in Worker for 16.02s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 08:36:58,071 [INFO] [core.py:595] distributed.core: Event loop was unresponsive in Worker for 16.02s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/__init__.py:55: UserWarning: Dependency not satisfied, torchani.ase will not be available
  warnings.warn("Dependency not satisfied, torchani.ase will not be available")
2024-02-01 08:37:45,392 [WARNING] [__init__.py:5] root: Warning: importing 'simtk.openmm' is deprecated.  Import 'openmm' instead.
TIME changed dir: 0.0s
TIME changed dir: 0.0s
Generated 31 conformers. 
Removed 3 conformers. 
Generated 36 conformers. 
Removed 19 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                              | 0/28 [00:00<?, ?it/s][W BinaryOps.cpp:601] Warning: floor_divide is deprecated, and will be removed in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values.
To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor'). (function operator())
Optimising conformer:   4%|▊                     | 1/28 [00:03<01:33,  3.46s/it]Optimising conformer:   7%|█▌                    | 2/28 [00:05<01:04,  2.49s/it]Optimising conformer:  11%|██▎                   | 3/28 [00:07<00:55,  2.22s/it]Optimising conformer:  14%|███▏                  | 4/28 [00:09<00:50,  2.12s/it]using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'

Optimising conformer:   0%|                              | 0/17 [00:00<?, ?it/s][AOptimising conformer:  18%|███▉                  | 5/28 [00:11<00:48,  2.13s/it]Optimising conformer:  21%|████▋                 | 6/28 [00:13<00:46,  2.11s/it]Optimising conformer:  25%|█████▌                | 7/28 [00:15<00:44,  2.13s/it]
Optimising conformer:   6%|█▎                    | 1/17 [00:07<02:00,  7.55s/it][AOptimising conformer:  29%|██████▎               | 8/28 [00:18<00:49,  2.50s/it]Optimising conformer:  32%|███████               | 9/28 [00:20<00:44,  2.37s/it]
Optimising conformer:  12%|██▌                   | 2/17 [00:11<01:20,  5.38s/it][AOptimising conformer:  36%|███████▌             | 10/28 [00:23<00:41,  2.30s/it]Optimising conformer:  39%|████████▎            | 11/28 [00:25<00:38,  2.28s/it]
Optimising conformer:  18%|███▉                  | 3/17 [00:14<01:00,  4.30s/it][AOptimising conformer:  43%|█████████            | 12/28 [00:27<00:36,  2.25s/it]Optimising conformer:  46%|█████████▊           | 13/28 [00:29<00:33,  2.24s/it]Optimising conformer:  50%|██████████▌          | 14/28 [00:31<00:31,  2.22s/it]Optimising conformer:  54%|███████████▎         | 15/28 [00:34<00:28,  2.22s/it]Optimising conformer:  57%|████████████         | 16/28 [00:36<00:26,  2.25s/it]Optimising conformer:  61%|████████████▋        | 17/28 [00:38<00:23,  2.18s/it]Optimising conformer:  64%|█████████████▌       | 18/28 [00:40<00:21,  2.14s/it]Optimising conformer:  68%|██████████████▎      | 19/28 [00:42<00:19,  2.14s/it]Optimising conformer:  71%|███████████████      | 20/28 [00:44<00:16,  2.12s/it]Optimising conformer:  75%|███████████████▊     | 21/28 [00:46<00:14,  2.06s/it]Optimising conformer:  79%|████████████████▌    | 22/28 [00:48<00:12,  2.10s/it]
Optimising conformer:  24%|█████▏                | 4/17 [00:39<02:40, 12.36s/it][AOptimising conformer:  82%|█████████████████▎   | 23/28 [00:51<00:11,  2.32s/it]Optimising conformer:  86%|██████████████████   | 24/28 [00:53<00:09,  2.26s/it]
Optimising conformer:  29%|██████▍               | 5/17 [00:43<01:53,  9.48s/it][AOptimising conformer:  89%|██████████████████▊  | 25/28 [00:55<00:06,  2.19s/it]Optimising conformer:  93%|███████████████████▌ | 26/28 [00:57<00:04,  2.18s/it]Optimising conformer:  96%|████████████████████▎| 27/28 [00:59<00:02,  2.11s/it]
Optimising conformer:  35%|███████▊              | 6/17 [00:50<01:35,  8.67s/it][AOptimising conformer: 100%|█████████████████████| 28/28 [01:01<00:00,  2.09s/it]Optimising conformer: 100%|█████████████████████| 28/28 [01:01<00:00,  2.21s/it]

Optimising conformer:  41%|█████████             | 7/17 [00:53<01:07,  6.74s/it][A
Optimising conformer:  47%|██████████▎           | 8/17 [00:56<00:50,  5.60s/it][A
Optimising conformer:  53%|███████████▋          | 9/17 [01:01<00:43,  5.41s/it][A
Optimising conformer:  59%|████████████▎        | 10/17 [01:05<00:35,  5.05s/it][A
Optimising conformer:  65%|█████████████▌       | 11/17 [01:11<00:30,  5.17s/it][A
Optimising conformer:  71%|██████████████▊      | 12/17 [01:17<00:27,  5.51s/it][A
Optimising conformer:  76%|████████████████     | 13/17 [01:23<00:22,  5.50s/it][A
Optimising conformer:  82%|█████████████████▎   | 14/17 [01:29<00:17,  5.76s/it][A
Optimising conformer:  88%|██████████████████▌  | 15/17 [01:36<00:12,  6.24s/it][A
Optimising conformer:  94%|███████████████████▊ | 16/17 [01:42<00:05,  5.99s/it][A
Optimising conformer: 100%|█████████████████████| 17/17 [01:49<00:00,  6.54s/it][AOptimising conformer: 100%|█████████████████████| 17/17 [01:49<00:00,  6.47s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
2024-02-01 08:41:24,014 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Bonds with undefined stereochemistry are:
 - Bond 1 (atoms 1-2 of element (N-C)

/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
2024-02-01 08:41:25,805 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Bonds with undefined stereochemistry are:
 - Bond 1 (atoms 1-2 of element (N-C)

TIME Completed the molecule generation in 179.5s.
TIME Completed the molecule generation in 188.6s.
TIME changed dir: 0.0s
Generated 34 conformers. 
Removed 31 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/3 [00:00<?, ?it/s]Optimising conformer:  33%|███████▋               | 1/3 [00:04<00:09,  4.72s/it]Optimising conformer:  67%|███████████████▎       | 2/3 [00:08<00:04,  4.09s/it]Optimising conformer: 100%|███████████████████████| 3/3 [00:14<00:00,  4.89s/it]Optimising conformer: 100%|███████████████████████| 3/3 [00:14<00:00,  4.74s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
2024-02-01 09:04:53,170 - distributed.core - INFO - Event loop was unresponsive in Worker for 5.47s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 09:04:53,170 [INFO] [core.py:595] distributed.core: Event loop was unresponsive in Worker for 5.47s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 09:05:13,568 - distributed.core - INFO - Event loop was unresponsive in Worker for 4.95s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-02-01 09:05:13,568 [INFO] [core.py:595] distributed.core: Event loop was unresponsive in Worker for 4.95s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
TIME Completed the molecule generation in 69.8s.
TIME changed dir: 0.0s
TIME changed dir: 0.0s
Generated 19 conformers. 
Removed 11 conformers. 
Generated 13 conformers. 
Removed 10 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/8 [00:00<?, ?it/s]Optimising conformer:  12%|██▉                    | 1/8 [00:05<00:37,  5.40s/it]Optimising conformer:  25%|█████▊                 | 2/8 [00:09<00:27,  4.64s/it]Optimising conformer:  38%|████████▋              | 3/8 [00:11<00:16,  3.25s/it]Optimising conformer:  50%|███████████▌           | 4/8 [00:13<00:11,  2.76s/it]Optimising conformer:  62%|██████████████▍        | 5/8 [00:15<00:07,  2.49s/it]Optimising conformer:  75%|█████████████████▎     | 6/8 [00:18<00:05,  2.90s/it]Optimising conformer:  88%|████████████████████▏  | 7/8 [00:22<00:03,  3.06s/it]Optimising conformer: 100%|███████████████████████| 8/8 [00:23<00:00,  2.63s/it]Optimising conformer: 100%|███████████████████████| 8/8 [00:23<00:00,  2.99s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
TIME Completed the molecule generation in 71.5s.
TIME changed dir: 0.0s
Generated 7 conformers. 
Removed 0 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/7 [00:00<?, ?it/s]Optimising conformer:  14%|███▎                   | 1/7 [00:03<00:20,  3.42s/it]Optimising conformer:  29%|██████▌                | 2/7 [00:05<00:12,  2.59s/it]Optimising conformer:  43%|█████████▊             | 3/7 [00:07<00:09,  2.40s/it]Optimising conformer:  57%|█████████████▏         | 4/7 [00:10<00:07,  2.41s/it]Optimising conformer:  71%|████████████████▍      | 5/7 [00:12<00:04,  2.39s/it]Optimising conformer:  86%|███████████████████▋   | 6/7 [00:14<00:02,  2.31s/it]Optimising conformer: 100%|███████████████████████| 7/7 [00:16<00:00,  2.34s/it]Optimising conformer: 100%|███████████████████████| 7/7 [00:16<00:00,  2.42s/it]
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/3 [00:00<?, ?it/s]/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
Optimising conformer:  33%|███████▋               | 1/3 [00:05<00:10,  5.22s/it]Optimising conformer:  67%|███████████████▎       | 2/3 [00:07<00:03,  3.78s/it]/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
Optimising conformer: 100%|███████████████████████| 3/3 [00:10<00:00,  3.08s/it]Optimising conformer: 100%|███████████████████████| 3/3 [00:10<00:00,  3.42s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
TIME Completed the molecule generation in 47.2s.
TIME changed dir: 0.0s
Generated 46 conformers. 
Removed 35 conformers. 
TIME Completed the molecule generation in 134.0s.
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                              | 0/11 [00:00<?, ?it/s]Optimising conformer:   9%|██                    | 1/11 [00:04<00:44,  4.45s/it]Optimising conformer:  18%|████                  | 2/11 [00:06<00:25,  2.86s/it]Optimising conformer:  27%|██████                | 3/11 [00:08<00:19,  2.39s/it]Optimising conformer:  36%|████████              | 4/11 [00:10<00:17,  2.54s/it]Optimising conformer:  45%|██████████            | 5/11 [00:12<00:13,  2.32s/it]Optimising conformer:  55%|████████████          | 6/11 [00:14<00:10,  2.18s/it]Optimising conformer:  64%|██████████████        | 7/11 [00:19<00:12,  3.10s/it]Optimising conformer:  73%|████████████████      | 8/11 [00:21<00:08,  2.82s/it]Optimising conformer:  82%|██████████████████    | 9/11 [00:25<00:05,  2.98s/it]Optimising conformer:  91%|███████████████████  | 10/11 [00:27<00:02,  2.75s/it]Optimising conformer: 100%|█████████████████████| 11/11 [00:29<00:00,  2.54s/it]Optimising conformer: 100%|█████████████████████| 11/11 [00:29<00:00,  2.68s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-combo1/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
2024-02-01 09:26:17,786 - distributed.utils_perf - INFO - full garbage collection released 14.84 MiB from 224986 reference cycles (threshold: 9.54 MiB)
2024-02-01 09:26:17,786 [INFO] [utils_perf.py:198] distributed.utils_perf: full garbage collection released 14.84 MiB from 224986 reference cycles (threshold: 9.54 MiB)
slurmstepd: error: *** JOB 19912196 ON sb006 CANCELLED AT 2024-02-01T09:54:47 ***
